scripts/validate-artifacts.ps1

```diff
@@ -85,7 +131,7 @@
     }
 } else {
     Write-Output `u{1F914}
-    throw "Unable to find Network Interface resouce. Please make sure it was migrated to the correct resource group and try again."
+    throw "Unable to find Network Interface resouce. Please re-deploy the VM and try again."
```

Comment:
The script checks for a Network Interface resource. Ensure that the resource type and properties being checked match the actual deployment to avoid false negatives.